browser/resources/settings/sources.gni

```diff
@@ -181,6 +181,7 @@ brave_settings_ts_extra_deps =
 brave_settings_mojo_files = [
   "$root_gen_dir/brave/components/ai_chat/core/common/mojom/settings_helper.mojom-webui.ts",
   "$root_gen_dir/brave/components/ai_chat/core/common/mojom/ai_chat.mojom-webui.ts",
+  "$root_gen_dir/brave/components/ai_chat/core/common/mojom/tab_tracker.mojom-webui.ts",
```

Comment:
This doesn't seem right, why is this in `browser/resources/settings`?

Reply:
This is required - the settings page uses the same mojom interfaces for configuration (clearing history, storing history, adding models) and the `ai_chat` mojom file needs to be able to import the `tab_tracker` one, even though the settings page isn't going to use the tab_tracker.

Reply:
I see, I wonder if we can do something to split these out in a follow-up. Can you add comments for now?